src/test/java/org/gridsuite/network/map/NetworkMapControllerTest.java

```diff
@@ -1986,19 +1986,7 @@ public void shouldReturnLineWhenGetBranchOrThreeWindingsTransformer() throws Exc
     }
 
     @Test
-    void shouldReturn2WTWhenGetBranchOrThreeWindingsTransformer() throws Exception {
```

Comment:
this test is deleted and case of getting voltage levels for a given branch/3WT is treated in an other PR